packages/mdast/mail-templates/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@republik/mdast-mail-templates",
+  "version": "1.0.0",
+  "description": "render mdast trees to email",
+  "private": true,
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    },
+    "./email": {
```

Comment:
I don't see the source file for this, we can remove it, no?

Reply:
excellent catch as always